### ISSUE TITLE
Signing

### ISSUE THREAD

**Issue by arrudagates** [InvArch/INV4-Git#8]
The client requires a system for signing extrinsics to be implemented, and for that the following flow is proposed:

1.  When called, the remote helper will look for the encrypted private key json file in a standard directory (custom directory generated for gitarch) and if it can't find the user's encrypted private key it will ask for the user to provide it, then save it in said directory.

2. When the remote helper needs to sign extrinsics, it prompts the user via CLI to provide the decryption password for their private key, just like when using the https helper with password authentication.

So in short this would be a CLI version of what the polkadot.js extension does for signing extrinsics.

Also make sure the storage is secure.
